### PR TITLE
Home feed

### PR DESCRIPTION
**Pull request by ntq-minhtu**

### CODE REVIEW

File: src/router/index.js [hunk 1]
```diff
@@ -24,6 +25,11 @@ export default new Router({
       path: '/live',
       name: 'live',
       component: Live
+    },
+    {
+      path: '/feed',
```
Comment (furuhiro-jp) [inline src/router/index.js:31]: I thought that the path on this screen would be the root.
Is not it better?

File: src/store/state.js [hunk 2]
```diff
@@ -1,6 +1,43 @@
...
-  noteList: [],
-  curriculumList: []
+  noteList: [
+    { title: 'カリキュラムタイトル', thumbUrl: './static/img/student.jpg', memo: 'フリーフォーマットでのテキストフリーフォーマットでのテキストフリーフォーマットでのテキストフリーフォーマットでのテキストフリーフォーマットでのテキストフリーフォーマットでのテキストフリーフォーマットでのテキスト', date: '2018.02.10' },
+    { title: 'カリキュラムタイトル', thumbUrl: './static/img/student.jpg', memo: 'フリーフォーマットでのテキストフリーフォーマットでのテキストフリーフォーマットでのテキストフリーフォーマットでのテキストフリーフォーマットでのテキストフリーフォーマットでのテキストフリーフォーマットでのテキスト', date: '2018.02.10' },
+    { title: 'カリキュラムタイトル', thumbUrl: './static/img/student.jpg', memo: 'フリーフォーマットでのテキストフリーフォーマットでのテキストフリーフォーマットでのテキストフリーフォーマットでのテキストフリーフォーマットでのテキストフリーフォーマットでのテキストフリーフォーマットでのテキスト', date: '2018.02.10' },
+    { title: 'カリキュラムタイトル', thumbUrl: './static/img/student.jpg', memo: 'フリーフォーマットでのテキストフリーフォーマットでのテキストフリーフォーマットでのテキストフリーフォーマットでのテキストフリーフォーマットでのテキストフリーフォーマットでのテキストフリーフォーマットでのテキスト', date: '2018.02.10' },
+
+  ],
+  curriculumList: [
```
Comment (furuhiro-jp) [inline src/store/state.js:11]: Should I change the curriculum list screen to get data from the store state as well?

https://github.com/dank1ntq/app/blob/1d44c004228c8159813aa1c11832527f7903f575/src/components/CurriculumList.vue#L54-L75

File: src/router/index.js (hunk 1 above)
Comment (ntq-minhtu) [reply src/router/index.js:31]: Yes, but i only use it temporarily, after integrating animation we'll not use router.

File: src/store/state.js (hunk 2 above)
Comment (ntq-minhtu) [reply src/store/state.js:11]: You should when you can do it

Comment (furuhiro-jp) [reply src/store/state.js:11]: I got it.
I will fix it with another PR. I have no time to say that ...

**Review by furuhiro-jp — APPROVED** [review]

**Pull request merged**